docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -111,13 +111,13 @@ The API authenticates via the [authentication backends](/administrator/configura
 
 Then a JWT token is returned and needs to be passed as a bearer in the headers for all the API calls.
 
-**Wrong credentials**: there is currently no error on bad authentication but instead a 302 redirect to the login page, e.g.:
+**Wrong credentials**: on bad authentication, it will return a `401 unauthorized` response, e.g.:
```

Comment:
Nice, indeed it is cleaner now with the new API

Reply:
Agreed!